cmd/root.go

```diff
@@ -48,7 +50,6 @@ func getRootCmd(args []string) *cobra.Command {
 		},
 	}
 
-	rootCmd.SetArgs(args)
 	rootCmd.PersistentFlags().AddGoFlagSet(flag.CommandLine)
```

Comment:
Just curious @jwendell - what's the purpose of this line? To pass through istio logging flags to Cobra's CLI? Or is there some other use case?

Reply:
TBH I don't remember/know why it's there.... I think I copied this from some Istio source code in the beginning...